pkg/datapath/linux/config/config.go

```diff
@@ -236,12 +236,12 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["SERVICE_NO_BACKEND_RESPONSE"] = "1"
 	}
 
-	if option.Config.EnableEncryptionStrictMode {
+	if option.Config.EnableEncryptionStrictModeEgress {
 		cDefinesMap["ENCRYPTION_STRICT_MODE"] = "1"
```

Comment:
Would it make sense to rename the egress-related variables also in the DP for readability?